models/articles.model.js

```diff
@@ -1,5 +1,6 @@
 const db = require('../db/connection');
 const format = require('pg-format');
+const { lastIndexOf } = require('../db/data/test-data/articles');
```

Comment:
What's going on here?? Could you please explain for me?

Reply:
@dentednerd  Oops I think it's automatically injected whilst I was saving..(?) I'll remove this.